rollup.config.js

```diff
@@ -0,0 +1,64 @@
+import commonjs from '@rollup/plugin-commonjs';
+import css from 'rollup-plugin-css-only';
+import vue from 'rollup-plugin-vue';
+import svg from 'rollup-plugin-vue-inline-svg';
+import esbuild from 'rollup-plugin-esbuild';
+import resolve from '@rollup/plugin-node-resolve';
+import filesize from 'rollup-plugin-filesize';
+import alias from '@rollup/plugin-alias';
+import { visualizer } from 'rollup-plugin-visualizer';
+import del from 'rollup-plugin-delete';
+
+// PostCSS plugins
+import tailwindcss from 'tailwindcss';
+import autoprefixer from 'autoprefixer';
+import assets from 'postcss-assets';
+
+module.exports = {
+  input: {
+    index: 'src/index.js',
+    ChecLineChart: 'src/components/ChecLineChart.vue',
+    ChecWysiwyg: 'src/components/ChecWysiwyg.vue',
```

Comment:
Why must these two components be listed individually? Could you add a comment to say why?

Reply:
Yeah it helps with code splitting on the UI library, but I don't think it's an amazing solution. Have put a comment in, but I'll also raise an issue to improve this later.